website/pages/en/community.js

```diff
@@ -62,46 +61,41 @@ function Community(props) {
         <translate>Channels:</translate>
         <Button
           target="_blank"
-          variant="dark"
           size="sm"
-          href="https://app.element.io/#/room/!HzySYSaIhtyWrwiwEV:matrix.org"
-          className="m-1"
+          href="https://matrix.to/#/#substrate-technical:matrix.org"
```

Comment:
I prefer using all the app.element.io links. If you try out the user experience. Using the app.element.io link, user is directly brought to the channel, while using matrix.io, one has to go thru three more clicks and making a lot of choices before able to enter the channel and chat.

Reply:
I prefer the opposite because I use the desktop client. The app link does not work for me, and I manually have to search for the room. Its far more work for anyone except element browser users. There are many more clients (and more coming) for matrix. This matrix.to link is "future proofing" for this and much better ux for everyone else